fetch_pyqt_windows.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Download and the PyQt4 Windows installer and unpack files from it into
+# pynsist_pkgs
+
+set -e
+
+PY_VERSION=3.4
+PYQT_VERSION=4.11.3
+QT_VERSION=4.8.6
+BITNESS=32
+
+INSTALLER_FILE=PyQt4-${PYQT_VERSION}-gpl-Py${PY_VERSION}-Qt${QT_VERSION}-x${BITNESS}.exe
+URL=http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-${PYQT_VERSION}/${INSTALLER_FILE}
+#wget -O "$INSTALLER_FILE" "$URL"
+
+rm -rf pyqt4-windows
+mkdir pyqt4-windows
+7z x -opyqt4-windows "$INSTALLER_FILE"
+
+rm -rf pynsist_pkgs
+mkdir pynsist_pkgs
+
+echo "Rearranging files into pynsist_pkgs..."
+mv 'pyqt4-windows/Lib/site-packages'/* pynsist_pkgs/
+rm pynsist_pkgs/PyQt4/assistant.exe pynsist_pkgs/PyQt4/designer.exe
+mv 'pyqt4-windows/$_OUTDIR/'*.pyd pynsist_pkgs/PyQt4/
+# These may not be necessary:
```

Comment:
Right... `qsci` is QScintilla, which we don't use, so it should be okay to omit it.

The `$_OUTDIR` thing had me scratching my head -- I couldn't see where that variable was being set. Upon realizing that it's a literal dollar-OUTDIR, then I figured it must be a `7z`-ism ;-)

I'm curious why `sip` went to `PyQt4/sip`.. `sip` needs to be importable as `import sip` from the top-level, and I'm assuming that this is already handled in the above site-packages `mv` command. What's in the sip directory?  If it's just small readmes/licenses then we can keep it and maybe keep it at `pynsist_pkgs/extras/sip` so that it's tucked away separately from the importable python modules.